### PR TITLE
Alps 576 hotspot texture update

### PR DESCRIPTION
**Pull request by rroux-gpsw**

### CODE REVIEW

**Comment by sw-team-release-gpsw** [comment]
[CLA assistant check](https://cla.gopro.com/sign) <br/>All committers have signed the CLA.

File: src/hotspots/Hotspot3D.js [hunk 1]
```diff
@@ -131,31 +148,37 @@ FORGE.Hotspot3D.prototype._parseConfig = function(config)
...
+        this._states.addConfig(config.states);
+    }
+
+    if (typeof config.transform === "object")
     {
         this._transform.load(config.transform);
     }
 
-    if (typeof config.animation !== "undefined")
+    if (typeof config.animation === "object")
```
Comment (aberthet-gpsw) [inline src/hotspots/Hotspot3D.js]: Beware, `typeof null = 'object'` (same below)

File: src/hotspots/HotspotStates.js [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+    {
+        if (this._onLoadComplete === null)
+        {
+            this._onLoadComplete = new FORGE.EventDispatcher(this);
+        }
+
+        return this._onLoadComplete;
+    }
+});
+
```
Comment (aberthet-gpsw) [inline src/hotspots/HotspotStates.js]: Mmmmh mmmh

File: reference/hotspot/hotspot.json [hunk 3]
```diff
@@ -38,6 +38,21 @@
...
+            "type": "boolean",
+            "title": "Interactive flag",
+            "description": "Is this hotspot is interactive and eligible to raycast",
+            "default": true
+        },
+
+        "cursor": {
+            "type": "string",
+            "title": "Pointer cursor",
+            "description": "The cursor to display when the pointer is over the hotspot.",
```
Comment (rroux-gpsw) [inline reference/hotspot/hotspot.json]: I add "Any valid css cursor value can be set."

File: src/hotspots/HotspotStates.js (hunk 2 above)
Comment (rroux-gpsw) [inline src/hotspots/HotspotStates.js]: Can you be more specific? :octocat:

File: src/hotspots/Hotspot3D.js (hunk 1 above)
Comment (rroux-gpsw) [inline src/hotspots/Hotspot3D.js]: changed for (typeof config.animation === "object" && config.animation !== null)

File: src/display/image/Image.js [hunk 4]
```diff
@@ -224,19 +224,6 @@ FORGE.Image.prototype._boot = function()
...
-    }
-    else if(this._renderMode === FORGE.Image.renderModes.CANVAS)
-    {
-        this._canvas = document.createElement("canvas");
-        this._canvas.style.position = "absolute";
-        this._canvas.style.top = "0px";
-        this._canvas.style.left = "0px";
-        this._dom.appendChild(this._canvas);
-    }
-
```
Comment (rroux-gpsw) [inline src/display/image/Image.js]: Yeah my bad, I broke the image in some cases, it's fixed (I hope so)

**Comment by aberthet-gpsw** [comment]
LGTM

**Pull request merged**